go/teams/chain.go

```diff
@@ -719,7 +733,7 @@ type checkInnerLinkResult struct {
 // Does not modify `prevState` but returns a new state.
 func (t *teamSigchainPlayer) addInnerLink(
 	prevState *TeamSigChainState, link *ChainLinkUnpacked, signer SignerX,
-	isInflate bool) (
+	isInflate bool, skipComputingHPrevInfo bool) (
```

Comment:
@mlsteele I'm adding this flag to addInnerLink because the inflate was messing up the computed values. Not super elegant but if we're not going to call addInnerLink out of order otherwise, it might be ok. What do you think?

Reply:
Seems like `skipComputingHPrevInfo == isInflate` so how about just use `isInflate`? That should hold as long as it remains impossible to inflate high links.

Reply:
Well, the issue I had was the player was run and lastKnownHprevs were computed. Then inflate was run on a middle link and it tried to compare the link's provided hPrevInfo with what was computed (which is accurate for *new* links only).

But yes, they are equal. Since we can maintain that addInnerLink is never called otherwise I'll use isInflate instead and add a comment.